Tools/scripts/generate_global_objects.py

```diff
@@ -284,6 +284,12 @@ def generate_runtime_init(identifiers, strings):
                 printer.write('')
                 with printer.block('.tuple_empty =', ','):
                     printer.write('.ob_base = _PyVarObject_IMMORTAL_INIT(&PyTuple_Type, 0)')
+        printer.write('')
+        with printer.block("static inline void\n_Py_StaticStrings_Intern(void)"):
+            printer.write(f'PyObject *string;')
+            for i in sorted(identifiers):
+                printer.write(f'string = &_Py_ID({i});')
+                printer.write(f'PyUnicode_InternInPlace(&string);')
```

Comment:
I think we can introduce a macro to half the generated function from ~600 to ~300 lines:

```suggestion
            printer.write('#define INTERN(id) \\')
            printer.write('    string = &_Py_ID(id); \\'
            printer.write('    PyUnicode_InternInPlace(&string);')
            for i in sorted(identifiers):
                printer.write(f'INTERN({i});')
            printer.write(f'#undef INTERN')
```

Reply:
Well hiding function call in a macro makes debugging harder and has hidden control flow, so I'll pass it out for now. Thanks!

Reply:
> makes debugging harder and has hidden control flow

Initially I missed this aspect, thank you for pointing out.